core/util/polyfill.js

```diff
@@ -38,17 +37,17 @@ if (typeof Object.assign != 'function') {
 }
 
 /* CustomEvent constructor (taken from MDN) */
-(function () {
-    function CustomEvent ( event, params ) {
+(() => {
+    function CustomEvent( event, params ) {
         params = params || { bubbles: false, cancelable: false, detail: undefined };
-        var evt = document.createEvent( 'CustomEvent' );
+        const evt = document.createEvent( 'CustomEvent' );
         evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
         return evt;
     }
 
     CustomEvent.prototype = window.Event.prototype;
 
-    if (typeof window.CustomEvent !== "function") {
+    if (typeof window.CustomEvent !== 'function') {
```

Comment:
This should probably also be disabled or we'll have a lot of noise once eslint is run.